util/string_util.h

```diff
@@ -16,7 +16,7 @@ extern std::vector<std::string> StringSplit(const std::string& arg, char delim);
 
 template <typename T>
 inline std::string ToString(T value) {
-#if !(defined OS_ANDROID) && !(defined CYGWIN)
+#if !(defined OS_ANDROID) && !(defined CYGWIN) && !(defined __FreeBSD__)
```

Comment:
Can you pls replace __FreeBSD__ with OS_FREEBSD? We #define OS_FREEBSD in our detection of target OS: https://github.com/facebook/rocksdb/blob/master/build_tools/build_detect_platform#L138